### ISSUE TITLE
Unit test test_Sound_voiceActivity.praat fails

### ISSUE THREAD

**Issue by rlaboiss** [praat/praat#1665]
The unit test test_Sound_voiceActivity.praat is failing with Praat version 6.1.41. It seems that file Example.wav is missing in the tarball. Here is how to trigger the bug:

```
$ cd dwtest
$ ../praat_nogui test_Sound_voiceActivity.praat
Error: Cannot open file “/var/data/rlaboiss/debian/PKGS/praat/praat/dwtest/Example.wav”.
Script line 11 not performed or completed:
« sound = Read from file: "Example.wav" »
Script “/var/data/rlaboiss/debian/PKGS/praat/praat/dwtest/test_Sound_voiceActivity.praat” not completed.
Praat: script command <<test_Sound_voiceActivity.praat>> not completed.
```

**Comment by rlaboiss** [comment]
Any news about this? This bug is preventing the building of the Debian package praat.

**Comment by davidweenink** [comment]
This error is also present in 6.1.42. 
Quick fix:  change the line 
sound = Read from file: "Example.wav"
to
sound = Read from file: "Example_vorbis_audio.ogg"
and the test will run correctly.
It will be corrected in 6.1.43

**Comment by rlaboiss** [comment]
Thanks for fixing this, David!
